jwst/flatfield/tests/test_g_average.py

```diff
@@ -0,0 +1,75 @@
+"""
+Test for flat_field.g_average
+"""
+import math
+
+import numpy as np
+
+from jwst.flatfield.flat_field import g_average
+
+def evaluate_polynomial(x, coeff):
+    """Evaluate a polynomial."""
+    n = len(coeff)
+    sum = np.zeros_like(x) + coeff[0]
+    for i in range(1, n):
+        sum = sum * x + coeff[i]
```

Comment:
Shouldn't the "*" and "+" operators be swapped in this expression, i.e. multiply x by coeff[i] and add the result to sum?

Reply:
No, but I do have the coefficients in an unusual order.  Here is an example of what I was thinking:

    sum = c[0]
    sum = c[0] * x + c[1]
    sum = (c[0] * x + c[1]) * x + c[2]
    sum = (c[0] * x**2 + c[1] * x + c[2]) * x + c[3]
           = c[0] * x**3 + c[1] * x**2 + c[2] * x + c[3]

I think this is consistent with the way I took the integral by modifying the coefficients and evaluating the polynomial, but I'll do some further checking.

Reply:
Oh, I see. OK, I guess that makes sense when it's spelled out in detail like that.